projects/kit/src/lib/masks/date-time/postprocessors/min-max-date-time-postprocessor.ts

```diff
@@ -27,9 +28,21 @@ export function createMinMaxDateTimePostprocessor({
         const [dateString, timeString] = parseDateTimeString(value, dateModeTemplate);
 
         if (!isDateTimeStringComplete(value, dateModeTemplate, timeMode)) {
+            const parsedDate = raiseSegmentValueToMin(
+                parseDateString(dateString, dateModeTemplate),
```

Comment:
**nitpicking:**

We use `parseDateString` and `parseTimeString` (with the the same args) two times in this file. Inside this `if`-condition and further inside `else`-condition.

Probably, we can move them outside `if/else` (raise it by few lines above).

Reply:
Same case with `createMinMaxDatePostprocessor`